app/code/Magento/Backend/etc/adminhtml/system.xml

```diff
@@ -503,22 +503,22 @@
                 <field id="base_url" translate="label comment" type="text" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
                     <label>Secure Base URL</label>
                     <backend_model>Magento\Config\Model\Config\Backend\Baseurl</backend_model>
-                    <comment>Specify URL or {{base_url}}, or {{unsecure_base_url}} placeholder.</comment>
+                    <comment>Specify URL or {{base_url}}{{unsecure_base_url}} placeholder.</comment>
```

Comment:
`Specify URL or {{base_url}}, or {{unsecure_base_url}} placeholder.` message must be displayed, do not remove "or". Fix all.

Reply:
@orlangur or is coming twice that the issue

Reply:
@orlangur  kindly look in to the issue #22169 you will to know the issue exactly

Reply:
@mohith227: @orlangur is correct, it should output `{{base_url}}` & `{{unsecure_base_url}}` in the comments instead of not showing those placeholders, and then it makes sense to leave the `or` in place. Issue #22169 is incorrectly defined, because the expected result is incorrect.

Reply:
@hostep or is their twice just check once

Reply:
@mohith227: please check the new expected result in #22169, I think it will make things clearer for you :)